apps/user-office-frontend/src/components/proposal/ProposalParticipant.tsx

```diff
@@ -23,18 +23,11 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 export default function ProposalParticipant(props: {
-  userId?: number;
-  userChanged: (user: BasicUserDetails) => void;
+  pi: BasicUserData | null | undefined;
```

Comment:
If it can be null or undefined should not we make it optional the handle that state in the component. We can also be more descriptive with the prop  name form pi to principalInvestigator.